service/worker/deployment/deployment_client.go

```diff
@@ -681,18 +680,7 @@ func (d *DeploymentClientImpl) updateWithStart(
 
 	policy := backoff.NewExponentialRetryPolicy(100 * time.Millisecond)
 	isRetryable := func(err error) bool {
-		if errors.Is(err, errRetry) {
-			return true
-		}
-		var multiErr *serviceerror.MultiOperationExecution
-		if errors.As(err, &multiErr) {
-			for _, e := range multiErr.OperationErrors() {
-				if common.IsServiceClientTransientError(e) {
-					return true
-				}
-			}
-		}
```

Comment:
This is not needed anymore.

Reply:
I think the history client will retry once, but since it's in IsService**Handler**RetryableError, the history server will retry once internally for each of those

Reply:
👍 I think so, too. And I think that's the desired behavior here.